zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -149,7 +149,7 @@ public boolean perNoteIsolated() {
     return ISOLATED.equals(perNote);
   }
 
-  public boolean isProcess() {
+  public boolean isSolated() {
```

Comment:
should `isSolated` be `isIsolated`? is that what you mean?

Reply:
Correct, it's a typo, I have fixed it